React/Views/RCTView.m

```diff
@@ -154,33 +156,58 @@ - (NSString *)accessibilityLabel
   return RCTRecursiveAccessibilityLabel(self);
 }
 
+-(void)setAccessibilityActions:(NSArray *)actions
+{
+  if (!actions || !actions.count) {
+    return;
+  }
+  _accessibilityActions = [NSMutableArray array];
+  accessibilityActionsNameMap = [[NSMutableDictionary alloc] init];
+  accessibilityActionsLabelMap = [[NSMutableDictionary alloc] init];
```

Comment:
```suggestion
  accessibilityActionsLabelMap = [NSMutableDictionary dictionary];
```

let's go with this style since we're using `[NSMutableArray array]` consistently in this file.